### ISSUE TITLE
Mac Os

### ISSUE THREAD

**Issue by FXUDarkNova** [phoboslab/jsmpeg#208]
What commands do you use to in mac to run the video and audio as iv got video struggling to get audio?

**Comment by dannysmc95** [comment]
What are you trying to do? Stream your Mac's video/audio?

**Comment by FXUDarkNova** [comment]
Yes video/audio from Mac iv got it working but audio is choppy and cutting out

**Comment by dannysmc95** [comment]
May I ask what you are trying to make? As the command I use has a zero latency setup, so there is no latency, but it's just not as clear...?

**Comment by FXUDarkNova** [comment]
Just want to stream my desktop with video/audio to multiple devices through the browser but audio is choppy
